interactive_engine/src/executor/store/src/v2/api/partition_snapshot.rs

```diff
@@ -66,18 +95,23 @@ pub trait PartitionSnapshot: Send + Sync {
         property_ids: Option<&Vec<PropertyId>>,
     ) -> Result<Records<Self::E>>;
 
+    /// Returns the out-degree of vertex `vertex_id` in `edge_relation`
     fn get_out_degree(
         &self,
         vertex_id: VertexId,
         edge_relation: &EdgeRelation,
     ) -> Result<usize>;
 
+    /// Returns the in-degree of vertex `vertex_id` in `edge_relation`
     fn get_in_degree(
         &self,
         vertex_id: VertexId,
         edge_relation: &EdgeRelation,
     ) -> Result<usize>;
 
+    /// Returns the `k`th out edge of vertex `vertex_id` in `edge_relation`.
+    ///
+    /// Properties of the edge are filtered by the `property_ids` optionally.
     fn get_kth_out_edge(
```

Comment:
Should get_kth_out_edge have edge_relation as one of its input pram?

Reply:
line 118